src/lib/containers/Menu.js

```diff
@@ -33,6 +35,21 @@ export default class Menu extends React.Component {
         }
     }
 
+    handleHover = (info) => (event) => {
+        if (!info.isSelected && event.target.tagName === "DIV") {
```

Comment:
I'd pull this logic into another function and share in hover enter/exit functions (in case it changes in the future)